app.json

```diff
@@ -1,15 +1,35 @@
 {
     "id": "ce0e318b-ffc0-4ce4-832b-f1b464beb22a",
-    "version": "0.0.1",
-    "requiredApiVersion": "^1.4.0",
+    "version": "0.0.2",
+    "requiredApiVersion": "^1.40.0",
     "iconFile": "icon.png",
     "author": {
-        "name": "Marcelo Schmidt",
-        "homepage": "https://github.com/marceloschmidt",
-        "support": "https://github.com/marceloschmidt/rocketchat-anonymizer/issues"
+        "name": "Rocket.Chat",
+        "homepage": "https://rocket.chat",
+        "support": "suporte@rocket.chat"
```

Comment:
Since the app is free and open source, are we still using the support email instead of the repo? If we are, there is a typo in the email :P

Reply:
Yes, let's use the email from support; we can add the repo within the technical documentation.